lib/src/analyzer_impl.dart

```diff
@@ -122,8 +125,13 @@ class AnalyzerImpl {
   void prepareErrors() {
     for (Source source in sources) {
       context.computeErrors(source);
+
       var sourceErrors = context.getErrors(source);
       errorInfos.add(sourceErrors);
+
+      if (options.strongMode) {
+        errorInfos.add(strongChecker.computeErrors(source));
```

Comment:
another thought here... I suspect inference will mess with the task model more than checking, and it's already implemented as a ResolverVisitor subclass: https://github.com/dart-lang/dev_compiler/blob/75337114c061c087d80cf78e2aa1a6a720ef072f/lib/src/checker/resolver.dart#L28 not sure if there's anything we can do to make it integrate better?

I didn't work much on it so I'm not super familiar with how it integrates into the resolution flow.